spec/struct.dd

```diff
@@ -181,23 +189,12 @@ static S q = { 1, 2 }; // q.a = 1, q.b = 2, q.c = 0, q.d = 7
 ------
 
         $(P Struct literals can also be used to initialize statics, but
-        they must be evaluable at compile time.)
+        they must be evaluatable at compile time.)
```

Comment:
A PR should not insert gratuitous errors to be fixed in a later PR, regardless of whether other pages do it. Please fix.